src/test/unit/views/case-reference-number-search-results.test.ts

```diff
@@ -0,0 +1,67 @@
+import {expect} from 'chai';
+import request from 'supertest';
+import sinon from 'sinon';
+import {app} from '../../../main/app';
+import fs from 'fs';
+import path from 'path';
+import {HearingRequests} from '../../../main/resources/requests/hearingRequests';
+import {request as expressRequest} from 'express';
+
+const searchTerm = '56-181-2097';
+const numOfResults = '1';
+const resultFound = '1 result successfully found';
+const PAGE_URL = `/case-reference-number-search-results?search-input=${searchTerm}`;
+
+const rowClass = 'govuk-table__row';
+
+let htmlRes: Document;
+
+const rawData = fs.readFileSync(path.resolve(__dirname, '../mocks/courtAndHearings.json'), 'utf-8');
+const allHearingData = JSON.parse(rawData);
```

Comment:
you dont seem to use allHearingData anywhere except to just extend below, can you combine the 2 instead of having unnecessary variable declaration?

Reply:
removed allHearingData.